### PR TITLE
set desktop title dynamically to focused app

### PR DESCRIPTION
**Pull request by NakulManchanda**
Signed-off-by: Nakul Manchanda <nakul.manchanda@ibm.com>

### CODE REVIEW

**Comment by NakulManchanda** [comment]
done as part of issue - https://github.com/zowe/zlux/issues/242

File: virtual-desktop/src/app/window-manager/mvd-window-manager/shared/window-manager.service.ts [hunk 1]
```diff
@@ -676,6 +680,17 @@ export class WindowManagerService implements MVDWindowManagement.WindowManagerSe
       return false;
     }
   }
+
+  setDesktopTitle(title?:String) {
+    const defaultTitle = 'Zowe Desktop';
```
[the remark this answers is not in the document]
Comment (NakulManchanda) [reply virtual-desktop/src/app/window-manager/mvd-window-manager/shared/window-manager.service.ts]: moved to top DEFAULT_DESKTOP_TITLE='Zowe'

**Comment by NakulManchanda** [comment]
hi, @DivergentEuropeans , @1000TurquoisePogs - can we have this in 1.7.0, i can add new item to have default title and title format configurable.
These changes seems to work well, let me know if there is any feedback.

This change will make title of Desktop Dynamic
- No Apps - Zowe Desktop
- App open and Focused - Zowe | <focused window title>

File: virtual-desktop/src/app/window-manager/mvd-window-manager/shared/window-manager.service.ts [hunk 2]
```diff
@@ -27,6 +27,8 @@ import { ContextMenuItem, Angular2PluginWindowActions,
 } from 'pluginlib/inject-resources';
 
 type PluginIdentifier = string;
+const DEFAULT_DESKTOP_SHORT_TITLE = 'Zowe';
+const DEFAULT_DESKTOP_TITLE = 'Zowe DESKTOP';
```
Comment (DivergentEuropeans) [inline virtual-desktop/src/app/window-manager/mvd-window-manager/shared/window-manager.service.ts]: Can you change this to 'Zowe Desktop'

Comment (NakulManchanda) [reply virtual-desktop/src/app/window-manager/mvd-window-manager/shared/window-manager.service.ts]: yes

**Pull request merged**